collector/Makefile

```diff
@@ -39,10 +39,7 @@ post-build:
 	cp -r "$(CMAKE_DIR)"/THIRD_PARTY_NOTICES/* container/THIRD_PARTY_NOTICES/
 	cp "$(LIBSINSP_BIN_DIR)/libsinsp-wrapper.so" container/libs/libsinsp-wrapper.so
 
-container/bin/collector:
-	$(MAKE) pre-build
-	$(MAKE) cmake-build/collector
-	$(MAKE) post-build
+container/bin/collector: pre-build cmake-build/collector post-build
```

Comment:
If we depend on the sequential build of the dependencies, we may want to add a `.NOTPARALLEL` statement to make sure that the behavior is correct even with `-j X`

https://www.gnu.org/software/make/manual/html_node/Parallel-Disable.html

Reply:
hmm excellent point, I'll add that

Reply:
It's even better this way. Out of curiosity, I wonder whether we could directly have
```
container/bin/collector:  cmake-build/collector
...
```